job-dsl-core/src/main/docs/examples/javaposse/jobdsl/dsl/helpers/publisher/PublisherContext/deployToWeblogic.groovy

```diff
@@ -0,0 +1,43 @@
+job('example') {
+    publishers {
+        deployToWeblogic {
+            // these are the default values used for optional fields, when not overridden by closure
+            mustExitOnFailure false
```

Comment:
I'm asking, because each time I change line 3 to ``deployToWeblogic() {`` I get a CodeNarc Error: UnnecessaryParenthesesForMethodCallWithClosure.